table/two_level_iterator.cc

```diff
@@ -154,14 +154,8 @@ void TwoLevelIterator::Prev() {
 
 
 void TwoLevelIterator::SkipEmptyDataBlocksForward() {
```

Comment:
Yes indeed. It was (from my debugging experience again) always the case. Adding an assert and re-running the tests is a good idea.